src/app/cluster/components/AddEditClusterModal/AddEditClusterForm.tsx

```diff
@@ -45,18 +45,38 @@ const valuesHaveUpdate = (values, currentCluster) => {
   if (!currentCluster || currentCluster.MigCluster.spec.isHostCluster) {
     return true;
   }
-
   const requireSSL = !currentCluster.MigCluster.spec.insecure;
-  const caBundle = currentCluster.MigCluster.spec.caBundle;
   const rawToken = atob(currentCluster.Secret.data.saToken);
   const existingEndpoint = currentCluster.MigCluster.spec.url;
-  const azureResourceGroup = currentCluster.MigCluster.spec.azureResourceGroup;
+
+  const getAzureResourceGroupVal = (azureResourceGroup) => {
```

Comment:
This looks dangerous. One branch returns a bool, the other returns a string. Should return something of the same type.

Reply:
For isAzure, the controller is expecting a boolean. The azureResourceGroup is a text input. Not sure if i am understanding the issue here?